modules/apps/commerce/commerce-product-asset-categories-web/src/main/java/com/liferay/commerce/product/asset/categories/web/internal/portlet/action/EditAssetCategoryCPAttachmentFileEntryMVCActionCommand.java

```diff
@@ -155,8 +155,8 @@ private void _updateCPAttachmentFileEntry(ActionRequest actionRequest)
 				displayDateMonth, displayDateDay, displayDateYear,
 				displayDateHour, displayDateMinute, expirationDateMonth,
 				expirationDateDay, expirationDateYear, expirationDateHour,
-				expirationDateMinute, neverExpire, titleMap, null, priority,
-				type, serviceContext);
+				expirationDateMinute, neverExpire, true, titleMap, null,
```

Comment:
Why true here?

Reply:
This is the category image, doesn't really matter. I set it to true since it is the default value.